src/website/app/demos/ButtonGroup/examples/uncontrolled.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import Button from '../../../../../library/Button';
+import ButtonGroup from '../../../../../library/ButtonGroup';
+
+export default {
+  id: 'uncontrolled',
+  title: 'Uncontrolled',
+  description: `Create an uncontrolled ButtonGroup by using the
+\`defaultChecked\` prop rather than the \`checked\` prop. Note that the
+\`defaultChecked\` prop may be set on the root (where the value may be a number
```

Comment:
Instead of "(where the value may be a number or an array of numbers representing the index or indices of the Button(s) to target, based on the...", maybe:

"(where the value may be a number representing the index of the Button to check or an array of those indices, based on the..."

Reply:
Yes, that's a bit clearer! Thanks.